benchmarks/bench_isolation_forest.py

```diff
@@ -1,33 +1,51 @@
-"""
-==========================================
-IsolationForest benchmark
-==========================================
-
-A test of IsolationForest on classical anomaly detection datasets.
-
-"""
-print(__doc__)
-
 from time import time
 import numpy as np
 import matplotlib.pyplot as plt
 from sklearn.ensemble import IsolationForest
 from sklearn.metrics import roc_curve, auc
 from sklearn.datasets import fetch_kddcup99, fetch_covtype, fetch_mldata
-from sklearn.preprocessing import LabelBinarizer
+from sklearn.preprocessing import MultiLabelBinarizer
 from sklearn.utils import shuffle as sh
 
-np.random.seed(1)
 
-datasets = ['http', 'smtp', 'SA', 'SF', 'shuttle', 'forestcover']
+def print_outlier_ratio(y):
+    """
+    Helper function to show the distinct value count of element in the target.
+    Useful indicator for the datasets used in bench_isolation_forest.py.
+    """
+    uniq, cnt = np.unique(y, return_counts=True)
+    print("----- Target count values: ")
+    for u, c in zip(uniq, cnt):
+        print("------ %s -> %d occurences" % (str(u), c))
+    print("----- Outlier ratio: %.5f" % (np.min(cnt) / len(y)))
+
 
+"""
+==========================================
+IsolationForest benchmark
+==========================================
+
+A test of IsolationForest on classical anomaly detection datasets.
+"""
+print(__doc__)
```

Comment:
You can put the `print(__doc__)` just below the import and before `print_outlier_ratio(y)`